fs/operations/operations.go

```diff
@@ -483,7 +484,14 @@ func Copy(ctx context.Context, f fs.Fs, dst fs.Object, remote string, src fs.Obj
 			break
 		}
 		// Retry if err returned a retry error
+		var retry bool
 		if fserrors.IsRetryError(err) || fserrors.ShouldRetry(err) {
+			retry = true
+		} else if t, ok := pacer.IsRetryAfter(err); ok {
+			time.Sleep(t)
```

Comment:
A debug message is probably a good idea here before the sleep as retry afters can be very long (5 minutes) and users get upset when things grind to a halt!

Something like this

```go
    fs.Debugf(src, "Sleeping for %v to obey Retry-After error: %v", t, err)
    time.Sleep(t)
```

Reply:
Updated.